### DISCUSSION TITLE
[Unstable] Refining with combined gene in right slot un-combines them

### DISCUSSION THREAD

**Discussion by Vykn** [FakeFishGames/Barotrauma#14647] [Bug reports]
### What happened?

If a combined gene is in the right slot when attempting to refine, the output gene will only be of the type that was in the left slot (essentially un-combining them)

![image](https://github.com/user-attachments/assets/a86eb607-9830-4316-b5af-fec50730b552)

![image](https://github.com/user-attachments/assets/77603489-6f2f-4364-ae12-8e6053594821)

By the way, I tested out the new system and I think it addresses basically every common complaint that people have had with it. I summarised [here](https://discord.com/channels/103209684680323072/668819422919524352/1285483759088566323). If there are any errors, that would be nice to know since I'd like to update the wiki page as soon as the update goes live and I don't have access to the source code in unstable.

**Comment by Regalis11** [comment]
This has been addressed since the report was created, looks like we just forgot to close the ticket. :)
